numerics/чебышёв_series.hpp

```diff
@@ -27,13 +27,23 @@ class ЧебышёвSeries {
   // The element at position i in |coefficients| is the coefficient of Tᵢ.  The
   // polynomials are scaled to the interval [t_min, t_max], which must be
   // nonempty.
-  explicit ЧебышёвSeries(std::vector<Vector> const& coefficients,
-                         Instant const& t_min,
-                         Instant const& t_max);
+  ЧебышёвSeries(std::vector<Vector> const& coefficients,
+                Instant const& t_min,
+                Instant const& t_max);
+  explicit ЧебышёвSeries(ЧебышёвSeries&& other);
```

Comment:
Why is the move constructor explicit? That means it's not MoveConstructible, so e.g. [`std::swap`](http://en.cppreference.com/w/cpp/algorithm/swap) doesn't work.